pkg/network/protocols/http/monitor.go

```diff
@@ -101,6 +105,17 @@ func (m *Monitor) Start() error {
 	return m.processMonitor.Initialize()
 }
 
+func (m *Monitor) GetUSMStats() map[string]interface{} {
+	if m == nil {
+		return map[string]interface{}{
+			"Error": USMStartupError.Error(),
```

Comment:
Shouldn't this just be an empty map?

Reply:
why empty map? I do want to have a reason when USM is not running